flight/goto/mission/goto.py

```diff
@@ -0,0 +1,59 @@
+"""
+File containing the move_to function responsible
```

Comment:
I think this file may have been copied into this folder by mistake, or left in the other folder, whichever the case, there is a duplicate.